packages/client/test/testdata/blocks/kaustinen4.json

```diff
@@ -1,6 +1,6 @@
 {
-  "353":{
```

Comment:
I have come across these random spacing changes myself with larger PR's that required several updates from master. I think it happens with each new verkle testnet update?

Reply:
I'm assuming so.  I didn't touch this file in my work.